openapi/jira.servicemanagement/Ballerina.toml

```diff
@@ -0,0 +1,12 @@
+[package]
+license = ["Apache-2.0"]
+keywords = ["Support//Customer Support", "Cost/Freemium"]
+org = "ballerinax"
+name = "jira.servicemanagement"
+distribution = "slbeta6"
+repository = "https://github.com/ballerina-platform/ballerinax-openapi-connectors"
+version = "0.1.0"
```

Comment:
Shall we update the version to 1.0.0 & build again

Reply:
Resolved